dom-manipulation/main.js

```diff
@@ -0,0 +1,26 @@
+let number = 0;
+
+const butt = document.querySelector('.hot-button');
+
+const clicks = document.querySelector('.click-count');
+
+butt.addEventListener('click', timesclicked);
+
+function timesclicked() {
+  number++;
+  clicks.textContent = 'Clicks: ' + number;
+  if (number < 4) {
+    butt.setAttribute('class', 'hot-button cold');
+  } else if (number < 7) {
+    butt.setAttribute('class', 'hot-button cool');
+  } else if (number < 10) {
+    butt.setAttribute('class', 'hot-button tepid');
+  } else if (number < 13) {
+    butt.setAttribute('class', 'hot-button warm');
+  } else if (number < 16) {
+    butt.setAttribute('class', 'hot-button hot');
+  } else {
+    butt.setAttribute('class', 'hot-button nuclear');
+    number = 0;
```

Comment:
You don't want to reset the number back to 0 here. You would just keep it going.